packages/expo-image/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Bumped iOS deployment target to 13.4. ([#25063](https://github.com/expo/expo/pull/25063) by [@gabrieldonadel](https://github.com/gabrieldonadel))
 - On `Android` bump `compileSdkVersion` and `targetSdkVersion` to `34`. ([#24708](https://github.com/expo/expo/pull/24708) by [@alanjhughes](https://github.com/alanjhughes))
+- Return a promise for `prefetch`. ([#25196](https://github.com/expo/expo/pull/25196) by [@gkasdorf](https://github.com/gkasdorf))
```

Comment:
In my opinion it's not a breaking change – users currently using the `prefetch` method will not have to do anything after upgrading (we don't take linter issues into account).
Let's move this to new features (because it extends the current behavior).

Reply:
I edited the changelog myself 😉